### ISSUE TITLE
Network Manager install

### ISSUE THREAD

**Issue by hjaltioj** [dlenski/network-manager-openconnect#5]
Hi

Is there a "simple" way to install this?

**Comment by dlenski** [comment]
Nope, not until GP support is merged into `openconnect` mainline, and this is merged into the NM mainline.

**Comment by antonisloizou** [comment]
Hey - you could at least list the dependencies though ;)

**Comment by zeph** [comment]
@antonisloizou 
```   sudo apt-get install intltool
   sudo apt install libgtk-3-dev
   sudo apt install libgcr-3-dev
   sudo apt install libsecret-1-dev
   sudo apt install libnm-util-dev libnm-glib-vpn-dev
   sudo apt install libnm-dev
```

**Comment by zeph** [comment]
just keep calling ./autogen.sh, it will tell you several times, iterating... what's missing
